open-metadata-resources/open-metadata-deployment/charts/vdc/values.yaml

```diff
@@ -91,74 +71,95 @@ image:
     pullPolicy: Always
 
   egeria:
-    repository: planetf1/egeria-egeriavdc
+    repository: localhost:5000/odpi/egeria
     tag: 1.1-SNAPSHOT
     pullPolicy: Always
 
   gaian:
-    repository: planetf1/gaian-egeriavdc
+    repository: localhost:5000/odpi/gaian-egeriavdc
     tag: 1.1-SNAPSHOT
     pullPolicy: Always
 
   atlas:
-    repository: planetf1/atlas-1773-egeria
-    tag: 1.1-SNAPSHOT
-    pullPolicy: Always
-
-  atlasinit:
-    repository: planetf1/atlas-init-egeria
+    repository: localhost:5000/odpi/apache-atlas
     tag: 1.1-SNAPSHOT
     pullPolicy: Always
 
   kafkamonitor:
-    repository: planetf1/kafka-monitor-egeriavdc
-    tag: 1.1-SNAPSHOT
-    pullPolicy: Always
-
-  egeriainit:
-    repository: planetf1/egeria-init-egeria
-    tag: 1.1-SNAPSHOT
-    pullPolicy: Always
-
-  igcproxyinit:
-    repository: planetf1/igcproxy-init-egeriavdc
-    tag: 1.1-SNAPSHOT
-    pullPolicy: Always
-
-  postgresqlinit:
-    repository: planetf1/postgresql-init-egeria
+    repository: localhost:5000/odpi/kafka-monitor-egeriavdc
     tag: 1.1-SNAPSHOT
     pullPolicy: Always
 
   postgresql:
-    repository: planetf1/postgresql-egeriavdc
-    tag: 1.1-SNAPSHOT
-    pullPolicy: Always
+    repository: postgres
+    tag: alpine
+    pullPolicy: IfNotPresent
 
   omrsmonitor:
-    repository: planetf1/omrsmonitor-egeriavdc
+    repository: localhost:5000/odpi/omrsmonitor-egeriavdc
     tag: 1.1-SNAPSHOT
     pullPolicy: Always
 
   ldapinit:
-    repository: planetf1/ldap-init-egeriavdc
+    repository: localhost:5000/odpi/ldap-init-egeriavdc
     tag: 1.1-SNAPSHOT
     pullPolicy: Always
```

Comment:
We should probably revert the pullPolicy (which I changed!) to if not present, also tag probably to latest. But this has implications on the current maven-oriented build due to the way tagging is done. Separate issue?

Reply:
Raised #812